concordium-node/src/consensus_ffi/ffi.rs

```diff
@@ -709,14 +714,37 @@ impl ConsensusContainer {
         ))
     }
 
-    pub fn in_baking_committee(&self) -> ConsensusIsInBakingCommitteeResponse {
-        wrap_c_committee_call!(self, |consensus| bakerIdBestBlock(consensus))
+    pub fn in_baking_committee(&self) -> (ConsensusIsInBakingCommitteeResponse, Option<u64>) {
+        let consensus = self.consensus.load(Ordering::SeqCst);
+
+        unsafe {
+            let mut baker_id: u64 = 0;
+            let mut has_baker_id: u8 = 0;
+
+            let result = bakerStatusBestBlock(consensus, &mut baker_id, &mut has_baker_id);
+            let status =
+                ConsensusIsInBakingCommitteeResponse::try_from(result).unwrap_or_else(|code| {
+                    panic!("Unknown Consensus Baking Committee FFI return code: {}", code)
```

Comment:
I'm not in favor of panics like this. Since this function is only used in rpc as far as I can tell I think a better approach would be to forward the error and then have the RPC respond with "InternalServerError ( error description )".

Reply:
If you want to keep the panic I would recommend using `unreachable!` instead since that documents the reason better.

Reply:
Went with `unreachable!`